deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/resource/Builds.java

```diff
@@ -210,11 +210,20 @@ public Response publish(
                 "Artifact URL points to unapproved location.");
         }
 
+        // Initialize the list of Source Control Managers
+        sourceControlManagerProxy.getSCMs();
```

Comment:
see the above comment in the "getSCM" function. then there is no need for this line. as the initialization is done in constructor.  In general, a getX function would expect the return value is being used, so just a getX statement here is odd.

Reply:
Yeah I see why the use of it is unclear. I will update accordingly.